<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbLoad">
       <property name="text">
        <string>1. Load picture --&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbToYUV">
       <property name="text">
        <string>2. Convert to YUV --&gt; </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbToRGB">
       <property name="text">
        <string>3. Convert to RGB using bug class</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="consoleContainer" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="4">
       <widget class="QGraphicsView" name="gvSourcePicture"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pbFitSource">
        <property name="text">
         <string>Fit image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="pbFitResult">
        <property name="text">
         <string>Fit Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pbResultZoomIn">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pbResultZoomOut">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="4">
       <widget class="QGraphicsView" name="gvResultPicture"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbSourceZoomOut">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pbSourceZoomIn">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pbStretchSource">
        <property name="text">
         <string>Streth image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="pbStretchResult">
        <property name="text">
         <string>Stretch Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
